sdlib/d/gc/capi.d

```diff
@@ -78,6 +78,16 @@ void __sd_gc_add_roots(const void[] range) {
 	gState.addRoots(range);
 }
 
+void __sd_gc_add_roots_ptr(const void* ptr, size_t length) {
```

Comment:
See the add TLS function, it is fine to pass a slice down, and we probably should for API consistency.

Reply:
Didn't we learn that it doesn't work? I can test anyway, and see what happens.

FWIW, extern(C) has no defined ABI for passing a dynamic array.

Reply:
Yep, it works.

I will change this.